internal/test_helpers/pass_all/your_program.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Check if git is working in PATH first
+if command -v git >/dev/null 2>&1; then
+    if [ "$1" = "write-tree" ]; then
+        git add .
+    fi
+    exec git "$@"
+fi
+
+# Find git binary in /tmp locations
+for tmpdir in /tmp/git-*/git; do
+    if [ -x "$tmpdir" ]; then
+        "$tmpdir" config --global init.defaultBranch main
+        "$tmpdir" config --global user.email "hello@codecrafters.io"
```

Comment:
If this is run on a dev's system directly, it's going to override their Git email and username. Any way to prevent this?